test/libsolidity/semanticTests/array/copying/copy_byte_array_in_struct_to_storage.sol

```diff
@@ -37,6 +37,13 @@ contract C {
 // compileViaYul: also
 // ----
 // f() -> 0x40, 0x80, 6, 0x6162636465660000000000000000000000000000000000000000000000000000, 0x49, 0x3132333435363738393031323334353637383930313233343536373839303120, 0x3132333435363738393031323334353637383930313233343536373839303120, 0x3132333435363738390000000000000000000000000000000000000000000000
+// gas ir: 176549
+// gas legacy: 174794
+// gas legacyOptimized: 174212
+// gas irOptimized: 172892
 // g() -> 0x40, 0xc0, 0x49, 0x3132333435363738393031323334353637383930313233343536373839303120, 0x3132333435363738393031323334353637383930313233343536373839303120, 0x3132333435363738390000000000000000000000000000000000000000000000, 0x11, 0x3132333435363738393233343536373839000000000000000000000000000000
+// gas ir: 106397
```

Comment:
Why is `gas irOptimized` missing here?

Reply:
See https://github.com/ethereum/solidity/pull/10863#discussion_r566840010